wordle/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>vārthekandpudip</title>
+    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+  <body>
+    <h1>angdlādyiza vārthekandpudip</h1>
+    <div id="gr"></div>
+    <div id="kb"></div>
+    <div id="er">Word isn't in the word list</div>
```

Comment:
translate this?

Reply:
Oh yeah, one sec I’ll do that

also I don’t know what Undardese for list is

Reply:
Done ✅ 
Lol I like how it says it’s now outdated